x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -138,16 +150,21 @@ const TGridStandaloneComponent: React.FC<TGridStandaloneProps> = ({
   indexNames,
   itemsPerPageOptions,
   onRuleChange,
+  onTotalItemsChange,
```

Comment:
looks like this is really the only function needed to be a new prop on timelines, the rest could be just used in the observability plugin